src/Docker/Configuration/Container.php

```diff
@@ -148,6 +148,74 @@ public function getConfigTreeBuilder()
                 ->end()
             ->end()
         ->end();
+
+        // artifacts
+        $root->children()
+            ->arrayNode('artifacts')
+            ->validate()
+                ->ifTrue(function ($v) {
+                    $enabledFilters = array_filter($v, function ($filter) {
+                        return isset($filter['enabled']) && $filter['enabled'] === true;
+                    });
+                    return count($enabledFilters) > 1;
+                })
+                ->thenInvalid('Multiple artifacts types, enable only one of "custom", "orchestration" or "runs".')
```

Comment:
Udělal jsem to tak, že může být zapnutý pouze jeden z těch typů filtru. Přišlo mi že to tak je správně.